NOTES.md

```diff
@@ -1,27 +1,18 @@
-# Release 2.0.0-RC1
-
-Released on 2015-03-23
-
-This is a release candidate for the upcoming RethinkDB 2.0 release.
-It is not for production use. It might still have bugs. If you decide
-to test it, please [back up all][backup-docs] your data first.
-
-Please report any bugs you find on GitHub
-(http://github.com/rethinkdb/rethinkdb/issues/new) or on our mailing
-list (https://groups.google.com/forum/#!forum/rethinkdb).
+# Release 2.0.0 (Tesuji)
 
 Release highlights:
 * Support for attaching a changefeed to the `get_all` and `union` commands
 * Improved support for asynchronous queries
```

Comment:
Should we mention production readyness here?
Something like
> Fully supported, production-ready release

@mglukhovsky might have a better idea for the phrasing.